openpgp/packet/signature.go

```diff
@@ -64,6 +64,7 @@ type Signature struct {
 	SigLifetimeSecs, KeyLifetimeSecs                        *uint32
 	PreferredSymmetric, PreferredHash, PreferredCompression []uint8
 	PreferredAEAD                                           []uint8
+	PolicyURI                                               []byte
```

Comment:
I know I mentioned this as an option previously, but do you have a strong opinion why this shouldn't be `string`? I think it would make more sense.